src/dmd/mars.d

```diff
@@ -2125,3 +2168,278 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
     return errors;
 }
 
+
+private __gshared bool includeImports = false;
```

Comment:
This random assortment of undocumented globals is more technical debt.

Reply:
This `boolean` and array represent all the data we need to know whether "compile imports" is enabled and all the patterns that were given.  Are you wanting them to be put in a struct as static variables?  What's the "debt" you're seeing that should be alleviated?